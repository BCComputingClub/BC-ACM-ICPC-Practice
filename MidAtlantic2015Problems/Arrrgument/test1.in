4 3
D 10000 9000 8000 6000
R 5000 500 50 5
S 500 500 50 50
Z 1 0 0 0
2 1
S 6000 5000
D 5000 4000
26 10
L 10 9 8 7 6 5 4 3 2 1 1
M 10 9 8 7 6 5 4 3 2 1 1
N 10 9 8 7 6 5 4 3 2 1 1
O 15 12 8 7 6 5 4 3 2 1 1
P 10 9 8 7 6 5 4 3 2 1 1
Q 10 9 8 7 6 5 4 3 2 1 1
R 10 9 8 7 6 5 4 3 2 1 1
S 10 9 8 7 6 5 4 3 2 1 1
T 15 12 8 7 6 5 4 3 2 1 1
U 10 9 8 7 6 5 4 3 2 1 1
V 10 9 8 7 6 5 4 3 2 1 1
W 10 9 8 7 6 5 4 3 2 1 1
X 10 9 8 7 6 5 4 3 2 1 1
Y 10 9 8 7 6 5 4 3 2 1 1
Z 10 9 8 7 6 5 4 3 2 1 1
A 10 9 8 7 6 5 4 3 2 1 1
B 10 9 8 7 6 5 4 3 2 1 1
C 15 12 8 7 6 5 4 3 2 1 1
D 10 9 8 7 6 5 4 3 2 1 1
E 10 9 8 7 6 5 4 3 2 1 1
F 15 12 8 7 6 5 4 3 2 1 1
G 10 9 8 7 6 5 4 3 2 1 1
H 10 9 8 7 6 5 4 3 2 1 1
I 10 9 8 7 6 5 4 3 2 1 1
J 15 12 8 7 6 5 4 3 2 1 1
K 10 9 8 7 6 5 4 3 2 1 1
26 10
L 10 9 8 7 6 5 4 3 2 1 1
M 10 9 8 7 6 5 4 3 2 1 1
N 10 9 8 7 6 5 4 3 2 1 1
O 15 12 8 7 6 5 4 3 2 1 1
P 10 9 8 7 6 5 4 3 2 1 1
Q 10 9 8 7 6 5 4 3 2 1 1
R 10 9 8 7 6 5 4 3 2 1 1
S 10 9 8 7 6 5 4 3 2 1 1
T 15 12 8 7 6 5 4 3 2 1 1
U 10 9 8 7 6 5 4 3 2 1 1
V 10 9 8 7 6 5 4 3 2 1 1
W 10 9 8 7 6 5 4 3 2 1 1
X 10 9 8 7 6 5 4 3 2 1 1
Y 10 9 8 7 6 5 4 3 2 1 1
Z 10 9 8 7 6 5 4 3 2 1 1
A 10 9 8 7 6 5 4 3 2 1 1
B 10 9 8 7 6 5 4 3 2 1 1
C 15 12 12 12 12 12 12 12 12 12 12
D 10 9 8 7 6 5 4 3 2 1 1
E 10 9 8 7 6 5 4 3 2 1 1
F 15 12 8 7 6 5 4 3 2 1 1
G 10 9 8 7 6 5 4 3 2 1 1
H 10 9 8 7 6 5 4 3 2 1 1
I 10 9 8 7 6 5 4 3 2 1 1
J 15 12 8 7 6 5 4 3 2 1 1
K 10 9 8 7 6 5 4 3 2 1 1
26 10
L 10 9 8 7 6 5 4 3 2 1 1
M 10 9 8 7 6 5 4 3 2 1 1
N 10 9 8 7 6 5 4 3 2 1 1
O 15 12 12 12 12 12 12 12 12 12 12
P 10 9 8 7 6 5 4 3 2 1 1
Q 10 9 8 7 6 5 4 3 2 1 1
R 10 9 8 7 6 5 4 3 2 1 1
S 10 9 8 7 6 5 4 3 2 1 1
T 15 12 8 7 6 5 4 3 2 1 1
U 10 9 8 7 6 5 4 3 2 1 1
V 10 9 8 7 6 5 4 3 2 1 1
W 10 9 8 7 6 5 4 3 2 1 1
X 10 9 8 7 6 5 4 3 2 1 1
Y 10 9 8 7 6 5 4 3 2 1 1
Z 10 9 8 7 6 5 4 3 2 1 1
A 10 9 8 7 6 5 4 3 2 1 1
B 10 9 8 7 6 5 4 3 2 1 1
C 15 12 12 12 12 12 12 12 12 12 12
D 10 9 8 7 6 5 4 3 2 1 1
E 10 9 8 7 6 5 4 3 2 1 1
F 15 12 8 7 6 5 4 3 2 1 1
G 10 9 8 7 6 5 4 3 2 1 1
H 10 9 8 7 6 5 4 3 2 1 1
I 10 9 8 7 6 5 4 3 2 1 1
J 15 12 8 7 6 5 4 3 2 1 1
K 10 9 8 7 6 5 4 3 2 1 1
26 10
L 10 9 8 7 6 5 4 3 2 1 1
M 10 9 8 7 6 5 4 3 2 1 1
N 10 9 8 7 6 5 4 3 2 1 1
O 15 12 8 7 6 5 4 3 2 1 1
P 10 9 8 7 6 5 4 3 2 1 1
Q 10 9 8 7 6 5 4 3 2 1 1
R 10 9 8 7 6 5 4 3 2 1 1
S 10 9 8 7 6 5 4 3 2 1 1
T 15 12 8 7 6 5 4 3 2 1 1
U 10 9 8 7 6 5 4 3 2 1 1
V 10 9 8 7 6 5 4 3 2 1 1
W 10 9 8 7 6 5 4 3 2 1 1
X 10 9 8 7 6 5 4 3 2 1 1
Y 10 9 8 7 6 5 4 3 2 1 1
Z 10 9 8 7 6 5 4 3 2 1 1
A 10 9 8 7 6 5 4 3 2 1 1
B 12 9 8 7 6 5 4 3 2 1 1
C 15 12 12 7 6 5 4 3 2 1 1
D 12 9 8 7 6 5 4 3 2 1 1
E 10 9 8 7 6 5 4 3 2 1 1
F 15 12 12 7 6 5 4 3 2 1 1
G 12 12 8 7 6 5 4 3 2 1 1
H 12 12 8 7 6 5 4 3 2 1 1
I 10 9 8 7 6 5 4 3 2 1 1
J 15 12 12 12 12 5 4 3 2 1 1
K 10 9 8 7 6 5 4 3 2 1 1
4 3
A 100 100 100 100
B 100 100 100 100
C 100 100 100 100
D 101 100 100 100
4 3
A 100 100 100 100
B 950 950 950 100
C 100 100 100 100
D 1000 900 900 900
4 3
A 100 100 100 100
B 950 950 950 950
C 100 100 100 100
D 1000 900 900 900
2 5
Z 100 90 80 70 60 40
Y 90 80 70 60 50 40
2 5
Z 10000 9000 8000 7000 7000 7000
Y 9000 8000 7000 6000 5000 4000
2 0
Q 10000
S 9000
0 0
