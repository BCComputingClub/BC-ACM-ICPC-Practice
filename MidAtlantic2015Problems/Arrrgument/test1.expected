R
D
C
F
C
C
D
B
D
Z
Y
Q
